sdk/spring/azure-spring-boot-test-aad/README.md

```diff
@@ -2,7 +2,50 @@
 
 ## Key concepts
 ## Getting started
+
+### How to run AADConditionalAccessIT. 
+
+#####  What is AAD Conditional Access?
+
+Azure Active Directory is a token-based authentication platform, [On-Behalf-Of flow] is the middle-tier service to make authenticated requests to the downstream service through token.
+[Conditional Access] is the tool used by Azure Active Directory to bring signals together, to make decisions, and enforce organizational policies. 
+In some cases, Conditional Access may cause the token get from middle-tier service was useless.![Policy Flow](docs/image-conditional-access-flow.png)
+
+##### Create applications and configure it.
+1. First, we need create three applications. The registered application name is ***webapp***, ***webapiA*** and ***webapiB***.![Application Name](docs/image-application-name.png)
```

Comment:
For the images, it's better to use sample account instead of blur the actual account.

Example: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-active-directory

Reply:
Got it.